apps/expo/package.json

```diff
@@ -126,7 +126,8 @@
     "eas-build-pre-install": "./scripts/pre-install.sh",
     "build:ios": "./scripts/build-ios.sh",
     "build:android": "./scripts/build-android.sh",
-    "build:dev-client:ios": "eas build --platform ios --profile development-simulator --non-interactive --local --output=\"./graysky.tar.gz\" --wait && eas build:run -p ios --path=\"graysky.tar.gz\"",
+    "build:dev-client:ios": "eas build --platform ios --profile development",
```

Comment:
Why not run it locally? Is this a linux limitation? Bare in mind it costs $2 per run

Reply:
Yes, the simulator doesn't run on linux afaik.

The EAS build service is free for hobbyists for up to 15 iOS build per month

Reply:
Ok fair enough

Reply:
(local builds don't work neither)